main/conceptual/getting-started/deploying.md

```diff
@@ -1,13 +1,11 @@
 # Deploying Smart Contracts
 
-The `agoric deploy` command in the [agoric command line tool](/guides/agoric-cli/commands.md#agoric-deploy) 
-supports deploying contracts and off-chain web applications that talk to them. The command 
-has two primary uses:
+The `agoric deploy` command in the [Agoric command line tool](/conceptual/agoric-cli/README.md#agoric-deploy) supports deploying contracts and off-chain web applications that talk to those contracts. The command has two primary uses:
```

Comment:
this link isn't relative? Why change the others to be relative, then?

Reply:
Well, it's semi-relative; it's not an absolute link. (i.e., https://docs.agoric.com/getting-started/before-using-agoric.html) I found both kinds of relative links already in the docs, (i.e., "/reference/repl/" and "../../reference/repl/") and they're both fine as far I'm concerned.